install/kubernetes/helm/Istio/charts/auth/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: auth 
+version: 0.1.0-1
+description: Helm chart for auth deployment
```

Comment:
How does this work? One chart per variant or one chart with variables ?

Reply:
So Istio is a parent chart, all components in charts folder are child charts with their own Chart.yam and values.yaml. values.yaml from parent can be used by child chart. At this moment I would prefer to have all variables in the parent's values.yaml but in structured way, for simplicity reasons. Let me know if I answered your question.